api/src/main/java/net/kyori/adventure/audience/ForwardingAudience.java

```diff
@@ -55,6 +60,30 @@
   @ApiStatus.OverrideOnly
   @NonNull Iterable<? extends Audience> audiences();
 
+  @Override
+  default @NonNull Audience filterAudience(final @NonNull Predicate<? super Audience> filter) {
+    @Nullable List<Audience> audiences = null;
+    for(final Audience audience : this.audiences()) {
+      if(filter.test(audience)) { // todo(kashike): should we test before filtering children?
```

Comment:
thoughts?

Reply:
Well I think we only want to filter children of audiences that match our predicate, so yeah, I say this is a good idea

Reply:
Does a forwarding audience have an identity of its own, or is a transparent wrapper around its children?

imo in the default implementation, a forwarding audience does not have any identity, so not testing is appropriate. Specific implementations of forwarding audiences may choose to add their own identity, in which case they can choose to implement testing themselves against the predicate in an overridden method.